nncf/utils.py

```diff
@@ -354,6 +354,15 @@ def should_consider_scope(scope_str: str, target_scopes: List[str], ignored_scop
                and not in_scope_list(scope_str, ignored_scopes)
 
 
+def get_wrapped_frozen_nncf_modules(model, ignored_scopes=None, target_scopes=None):
+    frozen_layers = []
+    for scope, module in model.get_nncf_modules().items():
+        if not module.weight.requires_grad:
+            if should_consider_scope(str(scope), target_scopes, ignored_scopes):
+                frozen_layers.append((scope, module))
+    return frozen_layers
```

Comment:
Please stop adding functions to utils when a proper place for them is inside `NNCFNetwork` (or any other specialized class, for that matter).

Reply:
just forgot to remove